two_factor/views/core.py

```diff
@@ -71,11 +73,17 @@ class LoginView(IdempotentSessionWizardView):
     }
 
     def has_token_step(self):
-        return default_device(self.get_user())
+        if self.token_required(self.request):
```

Comment:
This would read better if you make it a single statement: `return token_required(..) and default_device(..)`.